examples/undocumented/python/mkl_binclass.py

```diff
@@ -8,8 +8,9 @@
 
 #only run example if SVMLight is included as LibSVM solver crashes in MKLClassification
 try:
-	from shogun import SVMLight
-except ImportError:
+	from shogun import machine
+	machine("SVMLight")
```

Comment:
Let's remove if unused in the example.

Reply:
SVMLight needed for LibSVM solver in MKLClassification
https://github.com/shogun-toolbox/shogun/blob/ce5752287be7dc0eb826e26cd863eb3c395680f1/examples/undocumented/python/mkl_binclass.py#L9